src/bindings/python/tests/test_direct_comparison/test_shape.py

```diff
@@ -0,0 +1,21 @@
+from openvino import Shape, PartialShape
```

Comment:
The same: please, move these tests to [this file](https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/python/tests/test_graph/test_core.py#L112).